pkg/daemon/ceph/agent/flexvolume/attachment/resource.go

```diff
@@ -33,7 +34,7 @@ const (
 var VolumeResource = k8sutil.CustomResource{
 	Name:    CustomResourceName,
 	Plural:  CustomResourceNamePlural,
-	Group:   rookalpha.CustomResourceGroup,
+	Group:   rook.GroupName,
```

Comment:
IMO, having `CustomResource` provides some helpful context while reading, and to your point, so does `Name`. Maybe the longer `CustomResourceGroupName` is most explicit?

Reply:
Yeah I'll change it to `CustomResourceGroupName`